Mantle.xcodeproj/project.pbxproj

```diff
@@ -1238,9 +1238,11 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = CD7C6DB71D33AD72002EC294 /* watchOS-Framework.xcconfig */;
 			buildSettings = {
+				BITCODE_GENERATION_MODE = bitcode;
```

Comment:
This is not needed: https://github.com/Mantle/Mantle/pull/732#issuecomment-232623830